trade-finance-manager-ui/component-tests/case/facility/facility.component-test.js

```diff
@@ -52,6 +52,24 @@ describe(page, () => {
     wrapper.expectElement('[data-cy="facility-tenor"]').toExist();
   });
 
+  describe('when shouldDisplayFacilityEndDate is false', () => {
+    it('should not render facility end date', () => {
+      wrapper = render({ ...params, shouldDisplayFacilityEndDate: false });
+      wrapper.expectElement('[data-cy="is-using-facility-end-date"]').notToExist();
+      wrapper.expectElement('[data-cy="facility-end-date"]').notToExist();
+      wrapper.expectElement('[data-cy="bank-review-date"]').notToExist();
+    });
+  });
+
+  describe('when shouldDisplayFacilityEndDate is true', () => {
+    it('should display the correct default facility end date fields', () => {
```

Comment:
Could add one with BRD as well? Though think this is covered by e2e tests anyway

Reply:
added